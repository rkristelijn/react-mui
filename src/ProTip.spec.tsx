import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';

import ProTip from './ProTip';

jest.mock('./LightBulbIcon', () => 'LightBulbIcon');

describe('ProTip', () => {
  it('should initiate', () => {
    expect(ProTip).toBeTruthy();
  });

  it('should match snapshop', () => {
    const component = renderer.create(<ProTip />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
