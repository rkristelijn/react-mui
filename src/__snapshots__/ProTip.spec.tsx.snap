// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProTip should match snapshop 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-md368d-MuiTypography-root"
>
  <LightBulbIcon
    sx={
      Object {
        "mr": 1,
        "verticalAlign": "middle",
      }
    }
  />
  Pro tip: See more 
  <a
    className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
    href="https://mui.com/getting-started/templates/"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    templates
  </a>
   in the MUI documentation.
</p>
`;
